ibm_catalog.json

```diff
@@ -38,8 +38,16 @@
           "description": "Optional integration with [Cloud automation for Secrets Manager](https://cloud.ibm.com/catalog/7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3/architecture/deploy-arch-ibm-secrets-manager-6d6ebc76-7bbd-42f5-8bc7-78f4fabd5944-global) for centrally managing TLS certificates required by IBM MQ on Cloud."
         },
         {
-          "title": "Observability",
-          "description": "Optional integration with [Cloud automation for Observability](https://cloud.ibm.com/catalog/7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3/architecture/deploy-arch-ibm-observability-a3137d28-79e0-479d-8a24-758ebd5a0eab-global) services such as Log Analysis and Monitoring to support IBM MQ on Cloud instance monitoring and logging."
+          "title": "Sets up logging for the mq cloud instance",
+          "description": "Optionally, you can deploy Cloud automation for Cloud Logs to route, alert, and visualize platform logs that are generated by your mq cloud instance."
+        },
+        {
+          "title": "Sets up monitoring operational metrics for the mq cloud instance",
+          "description": "Optionally, you can deploy Cloud automation for Cloud Monitoring to measure how users and applications interact with your mq cloud instance."
```

Comment:
This should have a "Learn more" link to https://cloud.ibm.com/docs/mqcloud?topic=mqcloud-monitor_sysdig